test/loadtest/readme.md

```diff
@@ -0,0 +1,75 @@
+# Distributed load testing using Bedrock kubernetes Cluster and Locust
+
+Load testing is crucial to any application's success. We have chosen to use Locust as a tool for facilitating the load testing of the solution you have deployed in Bedrock. In order to begin load testing your application, make sure you have a cluster to deploy the Locust load tester on. If you do not have a load-testing cluster, follow the directions in [azure-simple](../../cluster/environments/azure-simple/) and use that cluster when you execute the following steps.
+
+## Setup
+
+1. Clone or download `Bedrock` git repo and navigate to `test/loadtest` folder.
+2. Update the `locust-tasks\tasks.py` file specific to however you would like your load test to be run. More details about how to write the tasks file can be found in the [Locust documentation](https://docs.locust.io/en/stable/writing-a-locustfile.html). All files in `locust-tasks` will be mounted to POD.
```

Comment:
Make a stronger statement here since most users will not have a "/productpage" route on their application that they need to change the file.

Reply:
Great point, updated the doc.